testing/shunit2/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: Olliver Schinagl <oliver@schinagl.nl>
+# Maintainer: Olliver Schinagl <oliver@schinagl.nl>
+pkgname=shunit2
```

Comment:
```suggestion
pkgname="shunit2"
```

Reply:
The irony, I'm usually the one that falls over this :) Sorry, I missed that.